pennylane/transforms/optimization/pattern_matching.py

```diff
@@ -91,9 +91,9 @@ def circuit():
     1: ──Z──╰Z─╭●─┤
     2: ──Z─────╰Z─┤
 
-    Note that with this pattern we also replace a ``pennylane.S``, ``pennylane.PauliZ`` sequence by ``pennylane.S``.
-    If one would like avoiding this, it possible to give a custom quantum cost dictionary with a negative cost for
-    ``pennylane.PauliZ``.
+    Note that with this pattern we also replace a ``pennylane.S``, ``pennylane.PauliZ`` sequence by
+    ``pennylane.Adjoint(pennylane.S)``. If one would like avoiding this, it possible to give a custom
```

Comment:
```suggestion
    ``Adjoint(S)``. If one would like avoiding this, it possible to give a custom
```
The `Adjoint` class is not available in the `pennylane` namespace, just `pennylane.ops.op_math`, as "adjoint" is the user facing constructor.  So maybe we should just write it out this way?

Reply:
Good idea